study-builder/studies/rarex/legacy.conf

```diff
@@ -49,80 +144,48 @@
           "shownExpr": null
         },
         {
-          "question": {
-            "questionType": "DATE",
-            "renderMode": "PICKLIST",
-            "fields": [
-              "DAY"
-            ],
-            "isRestricted": false,
-            "isDeprecated": false,
-            "hideNumber": true,
-            "additionalInfoHeaderTemplate": null,
-            "additionalInfoFooterTemplate": null,
-            "validations": [],
-            "displayCalendar": false,
-            "picklistConfig": null,
-            "stableId": "LOVED_PATIENT_DAY_OF_BIRTH",
-            "promptTemplate": {
-              "templateType": "TEXT",
-              "templateText": "",
-              "variables": []
-            },
-            "validations": []
+          "question" : ${subs.primary_diagnosis} {
+            "stableId": "LEGACY_PRIMARY_DIAGNOSIS"
           },
           "blockType": "QUESTION",
+          "blockGuid": null,
           "shownExpr": null
         },
         {
-          "question": {
-            "questionType": "DATE",
-            "renderMode": "PICKLIST",
-            "fields": [
-              "MONTH"
-            ],
-            "isRestricted": false,
-            "isDeprecated": false,
-            "hideNumber": true,
-            "additionalInfoHeaderTemplate": null,
-            "additionalInfoFooterTemplate": null,
-            "validations": [],
-            "displayCalendar": false,
-            "picklistConfig": null,
-            "stableId": "LOVED_PATIENT_MONTH_OF_BIRTH",
-            "promptTemplate": {
-              "templateType": "TEXT",
-              "templateText": "",
-              "variables": []
-            },
-            "validations": []
+          "titleTemplate": null,
+          "bodyTemplate": {
+            "templateType": "HTML",
+            "templateText": """<h4>$legacy_pass_away_date</h4>""",
+            "variables": [
+              {
+                "name": "legacy_pass_away_date",
+                "translations": [
+                  {
+                    "language": "en",
+                    "text": ${i18n.en.legacy_pass_away_date}
+                  }
+                ]
+              }
+            ]
           },
-          "blockType": "QUESTION",
+          "blockType": "CONTENT",
           "shownExpr": null
         },
         {
           "question": {
             "questionType": "DATE",
             "renderMode": "PICKLIST",
             "fields": [
-              "YEAR"
+              "DAY", "MONTH", "YEAR"
```

Comment:
Date of birth is usually `[month, day, year]`.

Reply:
That's what was in the ticket
<img width="403" alt="image" src="https://user-images.githubusercontent.com/71830706/116150075-9313b600-a6eb-11eb-9602-b5741e4c9e6e.png">

Reply:
👍  Got it.